### PR TITLE
feat!: remove centos7 and centos 8 stream templates

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
feat!: remove centos7 and centos 8 stream templates

These OSes are no longer supported

**Which issue(s) this PR fixes**:
Fixes 
https://issues.redhat.com/browse/CNV-48104

**Release note**:
```
feat!: remove centos7 and centos 8 stream templates

```

### CODE REVIEW

**Comment by nestoracunablanco** [comment]
@ksimon1 first of all, I think it's great that you're removing some distributions, especially those that don't support many architectures. However, I noticed that CentOS 6 is not being deleted. Is there a specific reason for this?

**Comment by ksimon1** [comment]
@nestoracunablanco yes, there is still business need for keeping the centos6 template in place.

File: automation/test-linux.sh [hunk 1]
```diff
@@ -80,7 +80,7 @@ if [[ $TARGET =~ opensuse.* ]]; then
   workloads=("server")
 fi
```
Comment (geetikakay) [inline automation/test-linux.sh:82]: since we are touching this file , should we cleanup centos6.*

File: generate-templates.yaml [hunk 2]
```diff
@@ -171,35 +142,6 @@
...
-      majorrelease: centos7
-      oslabels:
-       - centos7.0
-      osinfoname: "{{ oslabels[0] }}"
-      cloudusername: centos
-      containerdisk_urls: "{{ centos7_containerdisk_urls }}"
-      image_urls: "{{ centos7_image_urls }}"
-
   - name: Load CentOS 6 versions
     set_fact:
```
Comment (geetikakay) [inline generate-templates.yaml:146]: nit: maybe we can remove centos6?

**Pull request merged**